embassy-boot/boot/Cargo.toml

```diff
@@ -45,8 +45,11 @@ default_features = false
 features = ["rand", "std", "u32_backend"]
 
 [features]
+default = ["nightly"]
```

Comment:
In general we try to stay away from default features, because it's common that some library crate that depends on embassy-boot forgets to add `default-features=false` and then the end user has no way to disable them.

Could you remove it, and explicitly enable `nightly` in the examples? Thanks!